app/mailers/assignments_mailer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+class AssignmentsMailer < ActionMailer::Base
+  default from: 'transit-it@admin.umass.edu'
+
+  def upcoming_reminder(assignment, user)
```

Comment:
Couldn't you just call this with one argument, `assignment` and then set `@user` to be `@assignment.user`?